setup.py

```diff
@@ -0,0 +1,15 @@
+from distutils.core import setup
+
+setup(name='ha',
+      version='0.1',
+      description='High Availability for Cortx cluster',
+      url='https://github.com/Seagate/cortx-ha',
+      packages=[
+          'ha',
+          'ha/cli',
+          'ha/core',
```

Comment:
wanted to confirm
why do we need to add add sub dir 
'ha' will it not add recursively

Reply:
For development installation it is enough to have just `ha` directory (package).
However, for build distribution (bdist, bdist_rpm) just `ha` folder is not enough. All packages has to be specified.
I decided to make bdist/sdist feature functional since we may want to eventually upgrade this `setup.py` for distribution.